libkbfs/unflushed_path_cache.go

```diff
@@ -336,13 +368,50 @@ func (upc *unflushedPathCache) initialize(ctx context.Context,
 		default:
 		}
 
-		log.CDebugf(ctx, "Processing unflushed paths for %d items in "+
-			"the append queue", len(queue))
-		err := addUnflushedPaths(ctx, uid, key, codec, log, queue, cpp,
-			unflushedPaths)
-		if err != nil {
-			return nil, false, err
+		// Do the queued appends up to the first reinitialization.
+		for len(queue) > 0 {
+			var appends []unflushedPathMDInfo
```

Comment:
Any particular reason the queue can't be strictly FIFO? I know this is preserving existing behavior, but I was also wondering this in the previous UPC PRs.